routing/routing_session.hpp

```diff
@@ -34,13 +35,45 @@ class RouteMatchingInfo;
 
 namespace routing
 {
-struct SpeedCameraRestriction
+struct SpeedCameraOnRoute
 {
-  size_t m_index;  // Index of a polyline point where camera is located.
-  uint8_t m_maxSpeedKmH;  // Maximum speed allowed by the camera.
-
-  SpeedCameraRestriction(size_t index, uint8_t maxSpeed) : m_index(index), m_maxSpeedKmH(maxSpeed) {}
-  SpeedCameraRestriction() : m_index(0), m_maxSpeedKmH(std::numeric_limits<uint8_t>::max()) {}
+  SpeedCameraOnRoute() = default;
+  SpeedCameraOnRoute(double distFromBegin, uint8_t maxSpeedKmH)
+    : m_distFromBeginMeters(distFromBegin), m_maxSpeedKmH(maxSpeedKmH)
+  {}
+
+  // According to https://en.wikibooks.org/wiki/Physics_Study_Guide/Frictional_coefficients
+  // average friction of rubber on asphalt is 0.68
+  //
+  // According to: https://en.wikipedia.org/wiki/Braking_distance
+  // a =- \mu * g, where \mu - average friction of rubber on asphalt.
+  static auto constexpr kAverageFrictionOfRubberOnAspalt = 0.68;
+  static auto constexpr kGravitationalAcceleration = 9.80655;
+
+  // Used for calculating distance needed to slow down before a speed camera.
+  static double constexpr kAverageAccelerationOfBraking =
+    -kAverageFrictionOfRubberOnAspalt * kGravitationalAcceleration; // Metre per second squared.
```

Comment:
s/Metre/Metres/

Reply:
also, this file seems to be using the "Meters" form of spelling

Reply:
done

Reply:
left `meters`